packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { createDataProp } from '@leafygreen-ui/lib';
+import { ulStyleOverrides, LEFT_RIGHT_OFFSET } from './styles';
+
+export const sideNavGroupDataProp = createDataProp('side-nav-group-section');
```

Comment:
[question] I don't think this is actually being used, but I saw it in other components. Can this safely be removed?

Reply:
yes this can be removed if you aren't using the sideNavGroup selector, this prop is used to give us more specificity with css styles since we can't reference exact classNames